### PR TITLE
English Stringtable Update

### LINKED ISSUE #3046: [Bug]: Almost every grammar typo or antiquated description in addons/core/Stringtable.xml (ENGLISH TRANSLATION)
### Describe the bug

This is a compilation of most of the grammar errors, antiquated descriptions, and minor grammar nitpicks. I could find in the English version of the mod, and potential replacements to be more legible or indicative of what the error or event was. This will be formatted as a long list, with the offending piece being italicized and potential replacements displayed afterwards. This is my first time making an issue on GitHub, so let me know if there's a better way to do this. Thank you.

I was going to do every stringtable in one bug report, but I realized just how long this was going to be and figured its best to split this into multiple reports. This is the entirety of the core stringtable. I would be willing to make a PR for these changes on request, just be aware I have no way to test the PR.

### **addons/core/Stringtable.xml** ###

**Line 87:** STR_A3A_fn_ai_artySupport_noammo. 

Original: The battery has no ammo to fire. Reload it on HQ

Potential replacement: The mortar has run out of rounds. Wait a few minutes to restock it.

**Line 126:** STR_A3A_fn_ai_artySupport_selectOr

Original: You must select an artillery group or *it is a mobile mortar which is moving*.

Potential replacement: "the mortar team you have selected hasn't placed down the mortar yet"

**Line 409/415:** STR_A3A_fn_base_airspacecontrol_undercover_text, STR_A3A_fn_base_airspacecontrol_warning

Original: Unidentified helicopter&lt;br/&gt;&lt;br/&gt;You are closing in on the airspace of %1.&lt;br/&gt;&lt;br/&gt; Change your course or we will take defensive actions!

Potential replacement: Unidentified aircraft&lt;br/&gt;&lt;br/&gt;You are approaching the restricted airspace of %1.&lt;br/&gt;&lt;br/&gt; Change your course or we will take defensive actions!

Notes: This one has two strings, but they're the same and one appears to be obsolete based on a lack of translations.

**Line 679:** STR_A3A_fn_base_canmovehq_petros_pickedup

Original: *Petros currently* picked up.

Potential replacement: Petros is already

**Line 859:** STR_A3A_fn_base_flagaction_firex_fog

Original: I hate the fog

Potential replacement: Clear fog

**Line 885:** STR_A3A_fn_base_flagaction_firex_rain

Original: Rain rain go away

Potential replacement: Clear rain

**Line 961:** STR_A3A_fn_base_flagaction_intel_decifer

Original: Decifer Intel

Potential replacement: Decipher intel

Notes: I *think* this is obsolete? I've never seen it

**Line 1088:** STR_A3A_fn_base_flagaction_static_prevent

Original: Prevent *AIs using* this weapon

Potential replacement: AIs from using

**Line 1199:** STR_A3A_fn_base_hcdics_warning

Original: A Headless Client has been disconnected. This will cause malfunctions. Head back to HQ for saving ASAP and *ask and Admin* for a restart.

Potential replacement: ask an Admin

**Line 1248:** STR_A3A_fn_base_mrkwin_abort

Original: Aborted Outpost Capture

Potential replacement: Stop taking the flag

**Line 1302:** STR_A3A_fn_base_mrkwin_no_wl3

Original:  You cannot capture *Airports* until you reach War Level 3.

Potential replacement: airbases

**Line 1417**: STR_A3A_fn_base_outpdiag_no_radioman

Notes: obsolete?

**Line 1429**: STR_A3A_fn_base_outpdiag_no_resources

Original: You lack *of* resources to build this Watchpost or Roadblock. &lt;br/&gt;&lt;br/&gt; %1 HR and %2 € needed.

Potential replacement: the

**Line 1489**: STR_A3A_fn_base_placeselec_no_enemy_zone

Original: *Place selected is very close to enemy zones*.&lt;br/&gt;&lt;br/&gt; Please select another position.

Potential replacement: Position selected is very close to an enemy zone

**Line 1673**: STR_A3A_fn_base_sellveh_no_closer

Original: Vehicle must be closer than 50 meters to the *headquarters* marker.

Potential replacement: HQ

**Line 1782**: STR_A3A_fn_base_unlockveh_no_destr

Original: You cannot unlock/lock destroyed vehicles.

Potential replacement: You cannot lock or unlock destroyed vehicles.

**Line 1836**: STR_A3A_fn_base_unlockveh_no_owner2       SEE ABOVE

**Line 1912**: STR_A3A_fn_convoy_craiact_ammo_text

Original: A convoy from %1 is about to depart at %2. It will provide ammunition to %3. Try to intercept it. Steal or destroy that truck before it reaches *it's* destination.

Potential replacement: its

Notes: not on my watch

**Line 1936**: STR_A3A_fn_convoy_craiact_armored_text

Original: A convoy from %1 is about to depart at %2. It will reinforce %3 with armored vehicles. Try to intercept it. Steal or destroy *that thing* before it reaches it's destination.

Potential replacement: the armor

**Line 1984**: STR_A3A_fn_convoy_craiact_prison_text

Original: A group of POWs is being transported from %1 to %3, and it's about to depart at %2. *Try to intercept it. Kill or capture the truck driver to make them join you and bring them to HQ. Alive if possible.*

Potential replacement: Intercept the convoy and bring the POWs back to HQ, alive if possible.

**Line 2005**: STR_A3A_fn_convoy_craiact_reinf_text

Original: Reinforcements are being sent from %1 to %3 in a convoy, and it's about to depart at %2. *Try to intercept and kill all the troops and vehicle objective.*

Potential replacement: Intercept the convoy, kill all the troops, and destroy their vehicles. 

**Line 2029**: STR_A3A_fn_convoy_craiact_supply_text

Original: A truck with medical supplies destination %3 it's about to depart at %2 from %1. Steal that truck bring it to %3 and let people in there know it is %4 who's giving those supplies.

Potential replacement: A truck with medical supplies, destination %3, is about to depart at %2 from %1. Intercept the convoy, steal the truck, and bring it to %3, and let the people know %4 is giving the aid they need.

**Line 2200**: STR_A3A_fn_dialogs_ftradio_no_attack2

Original: You cannot Fast Travel to an area under attack or with enemies *in the surrounding*.

Potential replacement: nearby

**Line 2255**: STR_A3A_fn_dialogs_ftradio_no_ff

Original: You cannot fast travel while being *FF Punished*.

Potential replacement: punished for friendly fire

**Line 2360/2555/2585**: STR_A3A_fn_dialogs_minediag_no_radioman, STR_A3A_fn_dialogs_sqopt_radioman, STR_A3A_fn_dialogs_sqrec_radioman

Notes: Covered previously, obsolete?

**Line 2687**: STR_A3A_fn_init_cityinfo_overview_1

Original: %7&lt;br/&gt;&lt;br/&gt;Total pop: %1&lt;br/&gt;%6 Support: %2&lt;br/&gt;%5 Support: %3&lt;br/&gt;&lt;br/&gt;Murdered Pop: %4&lt;br/&gt;&lt;br/&gt;Click on the zone

Potential replacement: %7&lt;br/&gt;&lt;br/&gt;Total population: %1&lt;br/&gt;%6 Support: %2&lt;br/&gt;%5 Support: %3&lt;br/&gt;&lt;br/&gt;Murdered population: %4&lt;br/&gt;&lt;br/&gt;Click on a zone for detailed view.

**Line 3075**: STR_A3A_fn_intel_encr_err_2_err

Notes: Ilegible to Illegible

**Line 3633, 3666, 3682, 3698**: STR_A3A_fn_mission_conv_armor_text, STR_A3A_fn_mission_conv_prison_text, STR_A3A_fn_mission_conv_prison_text, STR_A3A_fn_mission_conv_supply_text

Notes: Changes to these have been noted above.

**Line 3713**: STR_A3A_fn_mission_des_ante_text

Original: We need to destroy or take a Radio Tower in %1. *This will interrupt %3 Propaganda Network*. Do it before %2.

Potential replacement: This will interrupt the coverage of %3 radio propaganda

**Line 3743**: STR_A3A_fn_mission_des_veh_text

Original: We know an enemy armor (%3) is stationed in %1. *It is a good chance to destroy or steal it before it causes more damage*. Do it before %2.

Potential replacement: We have a unique opportunity to steal or destroy the armor before it becomes a issue later on

**Line 3783**: STR_A3A_fn_mission_log_bank_hint_text2

Original: >Don't get the truck far from the *bank or count* will restart.

Potential replacement: bank, or the countdown

**Line 3807**: STR_A3A_fn_mission_log_bank_text

Original: We know Gendarmes are guarding a large amount of money in the bank of %1. Take this truck and go there before %2, hold the truck close to tha bank's main entrance for 2 minutes and the money will be transferred to the truck. Bring it back to HQ and the money will be ours.

Potential replacement: A squad of Gendarmeries are guarding a large amount of money in the bank of %1. Take the Box Truck and go there before %2. Once you're there, hold the truck next to the bank's main entrance for 2 minutes and the money will be transferred to the truck. Bring the truck back to HQ and the money will be ours.

**Line 3880**: STR_A3A_fn_mission_log_supply_text

Original: %1 population is in need of supplies. We may improve our relationship with that city if we are the ones who provide them. *I have placed a crate with supplies* near our HQ. Deliver the crate to %1 city center, hold it there for %3 minutes and it's done. Do this before %2.

Potential replacement: A crate with enough supplies for the whole town has been placed

**Line 4072**: STR_A3A_fn_orgp_memAdd_no_pointing

Original: You are not *pointing to anyone*.

Potential replacement: pointing at anyone

**Line 4272**: STR_A3A_fn_orgp_unitTraits_engi2

Original: Engineers do not have any bonus or penalties, but have the ability to use *Repair Kits* for vehicle repair.

Potential replacement: Toolkits

**Line 4418**: STR_A3A_fn_punish_oob_text

Original: Return to the AO before you are blown up, you have %1 seconds.

Potential replacement: Return to the AO ASAP! You have %1 seconds before you are blown to smithereens by enemy defenses.

**Line 4422**: STR_A3A_fn_punish_oob_titel

Original: Out of bounds

Potential replacement: We can't cover you there!

**Line 4492**: STR_A3A_fn_punish_punRel_enough

Original: Enough then.

Potential replacement: You have been released from the sea, think before you shoot next time.

**Line 4670**: STR_A3A_fn_reinf_addSqdHC_no_radio2

Notes: the infamous Radio Man strikes again

**Line 4758**: STR_A3A_fn_reinf_bombrun_increased

Original: Air Support increased in %1 points

Potential replacement: Available Air Support has been increased by %1 points.

**Line 4798**: STR_A3A_fn_reinf_bombrun_no_empty

Original: In order to convert, Vehicle must be empty.

Potential replacement: In order to convert a vehicle to Air Support, the vehicle must be empty.

**Line 5123**: STR_A3A_fn_reinf_dissSquad_no_nato
 
Notes: Even if it isn't obsolete, it needs factionGet(occ, "name")

**Line 5255**: STR_A3A_fn_reinf_NatoBomb_lack_supp

Original: You lack of enough Air Support to make this request

Potential replacement: You lack the Air Support points needed to fulfill this request.

**Line 5278**: STR_A3A_fn_reinf_NatoBomb_no_radio2 

Notes: our good friend Radio Man

**Line 5522**: STR_A3A_fn_revive_actRev_lost

Original: We lost %1.

Potential replacement: We lost %1. There's nothing you could have done, we need to keep fighting.

Notes: 5592 doesn't need changes

**Line 5550**: STR_A3A_fn_revive_actRev_no_dead

Original: %1 is already dead.

Potential replacement: %1 is already dead. Keep moving, soldier.

**Line 5710**: STR_A3A_fn_undercover_canGoUn_no_close

Original: You cannot go Undercover near *Airports*, Outposts, Seaports or Roadblocks

Potential replacement: Airbases

Notes: not sure if capitalization is warranted here

**Line 5731**: STR_A3A_fn_undercover_canGoUn_no_reason_naked

Original: %1&lt;br/&gt;Being naked. Thats what you think is unsuspicious?

Potential replacement: %1&lt;br/&gt;Being naked. Put some civilian clothes on, Petros can't bear to watch.

**Line 5808**: STR_A3A_fn_undercover_goUn_detect_airspace

Original: You have violated the airspace of %1!

Potential replacement: Unidentified aircraft, you have violated the restricted airspace of %1!

**Line 6012**: STR_A3A_garrison_error_enemies_near

Original: You can't recruit units *into garrison* when enemies are near the zone.

Potential replacement: into a garrison

**Line 6025**: STR_A3A_garrison_error_no_hr

Original: You lack HR to recruit units.

Potential replacement: You lack the HR necessary to recruit these units.

**Line 6076**: STR_A3A_garrison_exceed_limit

Original: Adding this squad to garrison will exceed garrison limit. Some of them will join garrison, the rest will be dismissed and their cost will be refunded.

Potential replacement: Adding this squad to the garrison will exceed the garrison limit. Some of the units will join the garrison, and the rest will be dismissed and their cost will be refunded.

**Line 6140**: STR_A3A_garrison_fail_not_markerzone

Original: You must select any %1 zone *with marker*.

Potential replacement: with a marker.

**Line 6179**: STR_A3A_garrison_fail_zone_lost

Original: %1 group has returned to HC control as the zone which was planned to be garrisoned has been lost.

Potential replacement: You have regained control of the %1 group as the zone where the group was going to be garrisoned has been lost.

**Line 6192**: STR_A3A_garrison_full_limit

Notes: it's "its", it is not "it's" :P

**Line 6274**: STR_A3A_GCTracker_tracker_notification

Original: Last Garbage Clean was %1 ago. Keep track of it. Forced GC in %2.

Potential replacement: Last Garbage Clean was %1 ago. Keep track of it. The server will automatically GC in %2.

**Line 6288**: STR_A3A_GCTracker_tracker_ran_gc

Original: Garbage Cleaner Tracker *run a garbage clean* because the time passed the threshold. Keep track of GC times.

Potential replacement: has ran a garbage clean

Notes: thanks Play3rgg on Discord

**Line 6363**: STR_A3A_keybinds_CfgUserAct_battleMenu_tip

Original: Opens the "Antistasis" battle menu.

Potential replacement: Antistasi

**Line 6387**: STR_A3A_keybinds_CfgUserAct_earPlugs_tip"

Original: Toggles the ear plugs (no effect when using *ace*)

Potential replacement: ACE

Notes: The French translator:

**Line 6399**: STR_A3A_keybinds_CfgUserAct_infoBar_tip

Original: Toggles the visibility of *Antistasis* information bar at the top of the screen.

Potential replacement: Antistasi's

Notes: it's me, hi, I'm the problem, it's me

**Line 6465**: STR_A3A_Params_bobChaosCrates_title

Original: [Experimental] *Truely* Random Crates: Remove all balance checks from Crates

Potential replacement: Truly

Notes: At tea time, everybody agrees

**Line 7352**: STR_antistasi_dialogs_AI_management_AI_control_tooltip
 
Original: Take personal control of the selected squad member or HC squad leader and be able to perform any kind of actions for 60 seconds. Control state will be cancelled if the player or the controlled unit receives any kind of damage

Potential replacement: Take control of a selected AI squad member or High Command Squad Leader. All actions can be performed through the AI for 60 seconds. You will lose control if the player or unit receives any damage

**Line 7379**: STR_antistasi_dialogs_AI_management_auto_heal_tooltip

Original: AI squad mates will *heal proactively* each other

Potential replacement: proactively heal

**Line 7493**: STR_antistasi_dialogs_AI_management_SITREP_tooltip

Original: Recover info about a HC squad status

Potential replacement: Gets a situation report from a selected HC squad. 

**Line 7518** STR_antistasi_dialogs_AI_management_squad_add_vehicle_tooltip

Original: Look at some vehicle and assign it to the selected squad *for it's use*

Potential replacement: for its use

Notes: I'll stare directly at the sun but never in the mirror

**Line 7686**: STR_antistasi_dialogs_carpet_bombing_add_tooltip

Original: Gain Airstrike points giving this aircraft to the faction Air pool

Potential replacement: Gain air support points by sacrificing the aircraft you're looking at. Use these points to call in HE, Cluster, and Napalm strikes.

**Line 7796**: STR_antistasi_dialogs_commander_comm_clean_tooltip

Original: Cleans several things in game. Use with caution as it freezes the mission

Potential replacement: Cleans dead bodies, dropped gear, destroyed vehicles and similar entities. Helps massively with client and server performance, and running this every hour is recommended.

**Line 7898**: STR_antistasi_dialogs_commander_comm_roadblock_delete_tooltip

Original: Remove selected observation post or roadblock, *money will be refunded*

Potential replacement: the money spent will be refunded

**Line 8108**: STR_antistasi_credits_generic_missiondesc_text

Original: Be a Resistance leader, be a hero.

Potential replacement: The Resistance needs leaders. Will you be the hero we need?

**Line 8134**: STR_antistasi_credits_generic_overview_text

Original: Build FIA Army from scratch and defeat the AAF and CSAT forces in a whole map Dynamic Mission.

Potential replacement: Build your rebellion from scratch and defeat the occupants and invading forces in a persistent whole map Dynamic Mission.

**Line 8671**: STR_antistasi_dialogs_hq_button_move_headquarters_tooltip

Original: Petros will join your group. Ammobox must be empty. To rebuild, select the Build action near Petros

Potential replacement: Petros will join your group. The arsenal box must be empty to move HQ. To rebuild, select the Build HQ here option on Petros.

**Line 8964**: STR_antistasi_dialogs_vehicle_purchase_civie_tooltip

Original: Civilian vehicles will allow you to travel in Undercover mode, while you stay close to roads and not entering bases, outposts and roadblocks.

Potential replacement: Civilian vehicles allow you to travel in Undercover mode. You will be spotted if  you stray too far from the road, as well as if you get too close to outposts, seaports, roadblocks, or airbases

**Line 9561**: STR_antistasi_dialogs_maps_spawn_distance_tooltip

Original: Affects performance. Please use this with caution. Set it to lower distances if you feel Antistasi is running *bad.*

Potential replacement: poorly

**Line 10168**: STR_antistasi_journal_entry_header_Features_10

Original: Ammobox transfer

Potential replacement: Arsenal transfer

**Line 10696**: STR_antistasi_journal_entry_text_AI_4

Original: AI Medic squad-mates will heal other AI’s automatically without the need to give any orders. If in combat, your AI will use smoke grenades for cover. If there is no medic in the group, the AI will use their own *medikits.*

Potential replacement: first aid kits

**Line 10708**: "STR_antistasi_journal_entry_text_AI_5

Original: Your AI squad-mates will automatically look for and scavenge any ammunition from nearby corpses, vehicles and ammoboxes if needed. Also they will try to renew their weapons with more powerful ones. If AI is inside a vehicle, they will loot weapons and launchers, and drop them in their truck until there is nothing in the area. Use this instead of the vanilla Rearm radio option.

Potential replacement: Your AI squad-mates will automatically look for and scavenge any ammunition from nearby corpses, vehicles, and loot boxes if needed. They will also try to improve their weapons if given the chance.  If AI is inside a vehicle, they will loot weapons and launchers, and drop them in their truck until there is nothing in the area. Use this instead of the vanilla Rearm radio option.

**Line 10732**: STR_antistasi_journal_entry_text_AI_7

Original: Select the unit and use the *'Dismiss Squad-mate'* option to send them back to HQ. You will recover some of the money you spent on them.

Potential replacement: 'Dismiss AI/Group'

**Line 10744**: STR_antistasi_journal_entry_text_commander_1

Original: Having a commander is necessary *because he is* the commander of the Faction attacking ground forces. Many options are available only to the commander and, depending on your settings, if you *play MP*, the most experienced player will usually occupy this position.

Potential replacements: because they are, play multiplayer

**Line 10780**: STR_antistasi_journal_entry_text_commander_12

Original: Use the radio and select a spot on the map with good *field of vision*. A group of snipers will join you. Send them to the selected place and they will spot for you, attempting to remain hidden and not engaging enemies upon contact.

Potential replacement: line of sight to enemy roads and markers

**Line 10828**: STR_antistasi_journal_entry_text_commander_2

Original: The Commander is responsible for assigning units as garrisons in conquered areas. Use the Garrison Management option in *the HQ flag* in order to add or remove garrison troops at each zone. Check Map Info screen for a quick overview of how many soldiers are assigned to each zone.

Potential replacement: Petros

**Line 10864**: STR_antistasi_journal_entry_text_commander_5

Notes: Same as above

**Line 10876**: STR_antistasi_journal_entry_text_commander_6

Original: Use the action menu on *your HQ Camp-fire* to rest for 8 hours.

Potential replacement: the tent at HQ

**Line 10959**: STR_antistasi_journal_entry_text_Default_1

Original: Welcome to Arma 3 - %1. This mission aims to simulate guerilla combat, tactics and strategic situations. It is not a quick and easy mod, it *is long term*, step-by-step mission with LOTS of features and enhanced AI.

Potential replacement: is a long term

**Line 10971**: STR_antistasi_journal_entry_text_Default_2

Original: You will win the game when most of the population *supports Faction* and you have conquered all the Airbases.

Potential replacement: supports the rebellion

**Line 10983**: STR_antistasi_journal_entry_text_Default_3

Original: You will lose the game if %1 has murdered 1/3 of the total population.

Potential replacement: You will lose the game if %1 murders 1/3 of the total population. To prevent this, respond to %1 punishment attacks against civilians.

**Line 10995**: STR_antistasi_journal_entry_text_Diary_1

Original: There is an All vs All war in %4. %1 and %2 are constantly attacking each other, but that does not mean they don't shoot at %3 on first sight.

Potential replacement: There is a total war on %4. The three factions competing for total control of the island are the rogue occupant military of %1, the ruthless invaders of %2, and the emerging rebellion of %3. %1 and %2 are constantly fighting, but also shoot %3 members on sight.

**Line 11055**: STR_antistasi_journal_entry_text_Diary_6

Original: Flag: Vehicle and Squad Unit Recruitment. Commander Options.&lt;br/&gt;Petros: Side-missions (Commander Only).&lt;br/&gt;Map: Game Options.&lt;br/&gt;Camp-fire: Rest for 8 hours and Clear Forest (Commander Only).

Potential replacement: Flag: Squad Unit Recruitment. Commander Options.&lt;br/&gt;Petros: Side-missions (Commander Only) and other actions related to managing HQ.&lt;br/&gt;Map: Game Options.&lt;br/&gt;Tent: Rest for 8 hours and Clear Forest (Commander Only).

**Line 11103**: STR_antistasi_journal_entry_text_Features_12

Original: Purchased static weapons won't despawn if you leave them in a base or zone, *AI garrison* will man them. Reinforce places as you desire.

Potential replacement: the AI garrison

**Line 11115**: STR_antistasi_journal_entry_text_Features_13

Original: You can build groups with up to 10 human players. Commander is excluded of this function.

Potential replacement: You can build groups with human players to organize different squads for different tasks.

**Line 11151**: STR_antistasi_journal_entry_text_Features_16

Original: Tanks and APC can be breached by engineers with explosives to open them and get their crew out. Check 'Breaching charges' for the charges needed. Breaching vehicles applies damage, so take care when breaching heavily damaged vehicles. To breach a vehicle, get close to it and select the breaching option!

Potential replacement: Armored vehicles can be breached by engineers with shaped charges to make the crew surrender and capture the vehicle for the rebellion. Look at the 'Breaching charges' section for a detailed description of what charges you need. If you are an engineer and have the charges, you simply need to approach the vehicle and select the breaching option.

**Line 11259**: STR_antistasi_journal_entry_text_Features_9

Original: Some zones, like airports require you to use the ""Take the Flag"" action on their respective flags in order to conquer them. Some others, like roadblocks and some outposts, simply require you to defeat the garrison.

Potential replacement: Once you have defeated the enemy garrison, you can go to the marker and select the "Take the Flag" option to convert the outpost to your faction.

**Line 11307**: STR_antistasi_journal_entry_text_gamemode_4

Original: All vs All war.&lt;br/&gt;&lt;br/&gt;%1 and %2 will attack each other and both to %3.

Potential replacement: All vs All war.&lt;br/&gt;&lt;br/&gt;%1 and %2 will attack each other, and both recognize the threat in the %3 rebellion and will shoot on sight.

**Line 11426**: STR_antistasi_journal_entry_text_SpecialK_2

Original: When you are the Commander, hit CTRL + SPACE to give way-points and other orders to your AI squads using the HC mode. *Non commanders may use it to check enemy contacts reported by the Faction communications network.*

Potential replacement: *Everything here should be cut*

**Line 11497**: STR_antistasi_journal_entry_text_tutorial_2

Original: Keep an eye on those. If it's too high, you will have to face more attacks with better troops.

Potential replacement: the enemy aggression on the Info Bar (default ALT + HOME to toggle)

**Line 11939, 11950**: STR_antistasi_dialogs_toggle_info_bar_body_off, STR_antistasi_dialogs_toggle_info_bar_body_on

Original: You've *turned on/off top information bar*. To enable it, press %1.

Potential replacement: turned on/off the top information bar

Notes: make sure to change both

**Line 12206**: STR_A3A_feedback_serverinfo_autostartwait

Original: Waiting for an admin to log in, or autostart timeout...

Potential replacement: Waiting for an admin to log in. If none join for 60 seconds, the server will load the most recent saved game.

**Line 12941**: STR_antistasi_dialogs_vehicle_manager_add_air_support_tooltip

Original: Gain Airstrike points giving this vehicle to the faction Air pool

Potential replacement: Gain air support points by sacrificing the aircraft you're looking at. Use these points to call in HE, Cluster, and Napalm strikes.

### How to reproduce

Open a vanilla instance of Antistasi and open the map screen for any of the map description issues, and use the appropriate method to reproduce any other messages.

### Version

3.4

### Have you altered the code?

No

### What i have changed

_No response_

### Map

n/A

### What server?

LAN Hosted

### Time bug occured (Server time/UTC)

n/A

### Mods

```html
n/A, vanilla Antitasi is sufficient
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Tiny-DM**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:

A lot of changes to addons/core/Stringtable.xml to improve clarity and grammar errors in certain places. Beyond a few specific grammar errors or typos, most of the changes were left to my purview and I've attempted to make them clearer for new players to understand. Let me know about any changes you think are necessary.

### Please specify which Issue this PR Resolves.
closes #3046 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
All of the "problematic strings"' have been tested (e.g. strings with specific formatting and breaks in them), the only strings left are the strings relating to Garbage Cleaning, members, and FF punishments because those aren't things I can test on a LAN host and I am far too smooth brain to go through the effort of setting up a dedicated server to test basic changes to strings. 

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -85,7 +85,7 @@
         <Chinesesimp>你当前的模式设定不支持这种类型的火力支援.</Chinesesimp>
       </Key>
       <Key ID="STR_A3A_fn_ai_artySupport_noammo">
-        <Original>The battery has no ammo to fire. Reload it on HQ.</Original>
+        <Original>The mortar has run out of rounds. Wait a few minutes to restock it.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:88]: not entirely sure if it reloads automatically or you actually have to send it to HQ

File: A3A/addons/core/Stringtable.xml [hunk 2]
```diff
@@ -857,7 +857,7 @@
         <Chinesesimp>释放战俘</Chinesesimp>
       </Key>
       <Key ID="STR_A3A_fn_base_flagaction_firex_fog">
-        <Original>I hate the fog</Original>
+        <Original>Clear fog</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:860]: Denied - please revert back

File: A3A/addons/core/Stringtable.xml [hunk 3]
```diff
@@ -883,7 +883,7 @@
         <Chinesesimp>清除附近的森林</Chinesesimp>
       </Key>
       <Key ID="STR_A3A_fn_base_flagaction_firex_rain">
-        <Original>Rain rain go away</Original>
+        <Original>Clear rain</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:886]: Denied - please revert back

File: A3A/addons/core/Stringtable.xml [hunk 4]
```diff
@@ -2027,7 +2027,7 @@
         <Chinesesimp>敌方增援车队</Chinesesimp>
       </Key>
       <Key ID="STR_A3A_fn_convoy_craiact_supply_text">
-        <Original>A truck with medical supplies destination %3 it's about to depart at %2 from %1. Steal that truck bring it to %3 and let people in there know it is %4 who's giving those supplies.</Original>
+        <Original>A truck with medical supplies, destination %3, is about to depart at %2 from %1. Intercept the convoy, steal the truck, and bring it to %3, and let the people know the %4 is giving the aid they need.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:2030]: steal the truck, and bring it to %3
-->
steal the truck, bring it to %3

File: A3A/addons/core/Stringtable.xml [hunk 5]
```diff
@@ -3741,7 +3741,7 @@
         <Czech>Zřícený vrtulník</Czech>
       </Key>
       <Key ID="STR_A3A_fn_mission_des_veh_text">
-        <Original>We know an enemy armor (%3) is stationed in %1. It is a good chance to destroy or steal it before it causes more damage. Do it before %2.</Original>
+        <Original>We know an enemy armor (%3) is stationed in %1. We have a unique opportunity to steal or destroy the armor before it becomes a issue later on. Do it before %2.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:3744]: becomes a issue
-->
becomes an issue

File: A3A/addons/core/Stringtable.xml [hunk 6]
```diff
@@ -3805,7 +3805,7 @@
         <Czech>Bankovní mise</Czech>
       </Key>
       <Key ID="STR_A3A_fn_mission_log_bank_text">
-        <Original>We know Gendarmes are guarding a large amount of money in the bank of %1. Take this truck and go there before %2, hold the truck close to tha bank's main entrance for 2 minutes and the money will be transferred to the truck. Bring it back to HQ and the money will be ours.</Original>
+        <Original>A squad of Gendarmeries are guarding a large amount of money in the bank of %1. Take the Box Truck and go there before %2. Once you're there, hold the truck next to the bank's main entrance for 2 minutes and the money will be transferred to the truck. Bring the truck back to HQ and the money will be ours.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:3808]: please remove the word "Box" from "Box Truck"

File: A3A/addons/core/Stringtable.xml [hunk 7]
```diff
@@ -4416,11 +4416,11 @@
         <Czech>Trest za střelbu do vlastních</Czech>
       </Key>
       <Key ID="STR_A3A_fn_punish_oob_text">
-        <Original>Return to the AO before you are blown up, you have %1 seconds.</Original>
+        <Original>Return to the AO ASAP! You have %1 seconds before you're shredded by enemy defenses.</Original>
         <Czech>Vraťte se do operační oblasti, než budete vyhozeni do povětří, máte %1 sekund.</Czech>
       </Key>
       <Key ID="STR_A3A_fn_punish_oob_titel">
-        <Original>Out of bounds</Original>
+        <Original>We can't cover you there!</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:4423]: Please revert, too long and not clear enough

File: A3A/addons/core/Stringtable.xml [hunk 8]
```diff
@@ -8669,7 +8669,7 @@
         <Chinesesimp>将总部转移到其他区域</Chinesesimp>
       </Key>
       <Key ID="STR_antistasi_dialogs_hq_button_move_headquarters_tooltip">
-        <Original>Petros will join your group. Ammobox must be empty. To rebuild, select the Build action near Petros</Original>
+        <Original>Petros will join your group. The arsenal box must be empty to move HQ. To rebuild, select the Build HQ here option on Petros.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:8672]: The inventory of the arsenal box must....

File: A3A/addons/core/Stringtable.xml [hunk 9]
```diff
@@ -10742,7 +10742,7 @@
         <Chinesesimp>选择单位并使用"解散班组成员"来将他们送回总部。你将回收部分开销。</Chinesesimp>
       </Key>
       <Key ID="STR_antistasi_journal_entry_text_commander_1">
-        <Original>Having a commander is necessary because he is the commander of the Faction attacking ground forces. Many options are available only to the commander and, depending on your settings, if you play MP, the most experienced player will usually occupy this position.</Original>
+        <Original>Having a commander is necessary because they are the commander of the Faction attacking ground forces. Many options are available only to the commander and, depending on your settings, if you play multiplayer, the most experienced player will usually occupy this position.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:10745]: Change "they are" back to "he is".

Comment (Bob-Murphy) [reply A3A/addons/core/Stringtable.xml:10745]: End review 17.11.2023 - continue after

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Stringtable.xml (hunk 1 above)
Comment (Tiny-DM) [reply A3A/addons/core/Stringtable.xml:88]: mortars resupply on resource ticks as long as a there is no player in it. teleporting it to hq just means you can hit the box

**Comment by Tiny-DM** [comment]
Fixed. Are 860 and 886 just for consistency?

**Comment by Bob-Murphy** [comment]
> Fixed. Are 860 and 886 just for consistency?

It's fun, known and already translated. No real need to change it.

File: A3A/addons/core/Stringtable.xml [hunk 10]
```diff
@@ -10826,7 +10826,7 @@
         <Chinesesimp>如果启用了服务器会员功能, 你可以在总部的旗帜上查询哪些服务器成员在线和非会员玩家的总数。</Chinesesimp>
       </Key>
       <Key ID="STR_antistasi_journal_entry_text_commander_2">
-        <Original>The Commander is responsible for assigning units as garrisons in conquered areas. Use the Garrison Management option in the HQ flag in order to add or remove garrison troops at each zone. Check Map Info screen for a quick overview of how many soldiers are assigned to each zone.</Original>
+        <Original>The Commander is responsible for assigning units as garrisons in conquered areas. Use the Garrison Management option in Petros in order to add or remove garrison troops at each zone. Check Map Info screen for a quick overview of how many soldiers are assigned to each zone.</Original>
```
Comment (Bob-Murphy) [inline A3A/addons/core/Stringtable.xml:10829]: wouldn't "on Petros" be better than "in Petros"?

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Stringtable.xml (hunk 10 above)
Comment (Tiny-DM) [reply A3A/addons/core/Stringtable.xml:10829]: It's inside the HQ management menu on Petros, so I'll change it to "Garrison Management option inside the HQ Management action on Petros." Does that work?

Comment (Tiny-DM) [reply A3A/addons/core/Stringtable.xml:10829]: Regardless, the entire in-game guide needs to be redone anyways. I fixed everything glaring (e.g. all the actions used to be on the flag/campfire) but after this I'll probably work on redoing that by just using the data on the wiki

Comment (Bob-Murphy) [reply A3A/addons/core/Stringtable.xml]: works for me

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**